junit-jupiter-engine/src/test/java/org/junit/jupiter/api/AssertInstanceOfAssertionsTests.java

```diff
@@ -93,7 +93,7 @@ private <T> void assertInstanceOfSucceeds(Class<T> expectedType, Object actualVa
 
 	private void assertInstanceOfFails(Class<?> expectedType, Object actualValue, String unexpectedSort) {
 		String valueType = actualValue == null ? "null" : actualValue.getClass().getCanonicalName();
-		String expectedMessage = String.format("Unexpected %s ==> expected: <%s> but was: <%s>", unexpectedSort,
+		String expectedMessage = String.format("Unexpected %s, expected: <%s> but was: <%s>", unexpectedSort,
```

Comment:
Please don't change the prefix delimiter to a comma.

This has been in place since 5.0 GA, and I don't think we should be changing the formatting this late in the game.

Reply:
Never mind. As you pointed out elsewhere, this is not a change of the prefix delimiter.